lib/modules/manager/npm/post-update/yarn.ts

```diff
@@ -223,9 +223,9 @@ export async function generateLockFile(
       } else {
         // `yarn up` updates to the latest release, so the range should be specified
         commands.push(
-          `yarn up ${lockUpdates
+          `yarn up -R ${lockUpdates
             // TODO: types (#7154)
-            .map((update) => `${update.depName!}@${update.newValue!}`)
+            .map((update) => `${update.depName!}`)
```

Comment:
shouldn't this packageName now? 🤔

Reply:
Maybe, but I prefer not to make this change now